src/services/bookingServices.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable max-len */
+import Model from '../models';
+
+const {
+  Booking, Accommodation, TripRequest, Room
+} = Model;
+
+/**
+ * Create a new item
+ * @param {Object} model
+ * @param {Object} data
+ * @param {Object} res
+ * @returns {Function} responseHandler
+ */
+export const create = async (model, data) => {
+  const trip = await TripRequest.findByPk(data.tripId);
+  if (!trip) {
+    return 'no-trip';
+  }
+  if (trip.status !== 'approved') {
+    return 'trip-invalid';
+  }
+  const accommodation = await Accommodation.findByPk(data.accommodationId);
+  if (!accommodation) {
+    return 'no-accommodation';
+  }
+  const result = await model.create(data);
+
+  return result;
+};
+
+/**
+ * @returns {object} response object containing the information of all rows
+ */
+export const findAllBookings = async () => {
+  const bookings = await Booking.findAll({
+    include: {
+      model: Accommodation,
+      as: 'accommodation',
+      attributes: ['name', 'country', 'city', 'address', 'type'],
+      include: {
+        model: Room,
+        as: 'rooms',
+        attributes: ['id', 'name', 'type']
+      }
+    }
+  });
+  return bookings;
+};
+
+/**
+ * @param {Number} userId
+ * @returns {object} response object containing the information of all bookings created by a specific user
+ */
+export const findAllBookingsWhere = async (userId) => {
```

Comment:
Don't you think `findUserBookings` seem more appropriate than`findAllBookingsWhere`?